src/bindings/python/constraints.txt

```diff
@@ -22,4 +22,4 @@ paddlepaddle==2.6.0
 tensorflow>=1.15.5,<2.18.0
 six~=1.16.0
 protobuf>=3.18.1,<4.0.0
-onnx==1.15.0
+onnx==1.16.0
```

Comment:
```suggestion
onnx==1.16.2
```

Reply:
https://github.com/openvinotoolkit/openvino/pull/25871

Reply:
Thanks for info, they had only 1.16.1 until yesterday :) I'll check how to do it better. Last time we discussed to move 1.16.0 and after that to 1.16.1 (now 1.16.2). JFYI.

Reply:
Decided to move to 1.16.2 in case of no new issues

Reply:
Huh, even conan doesn't have it right now...